### PR TITLE
fixed links

### PR DESCRIPTION
**Pull request by wolfdancer**
- Updated the link to blog post to the how-to article since the latter is more up-to-date
- Fixed another link to the API Doc

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3cce0b848a/how-to/).

File: content/rackspace-metrics/rackspace-metrics-overview.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ Rackspace Metrics consists of two components: **metrics ingestion** and
     Metrics supports metrics visualization in the Rackspace Cloud
     Control Panel and Rackspace Intelligence for data collected through
     Cloud Monitoring. In addition, Rackspace Metrics can also integrate
-    with Graphite and Grafana. See the following blog post for details:
-    <http://www.rackspace.com/blog/use-grafana-for-all-of-your-monitoring-data-rackspace-or-otherwise/>.
+    with Graphite and Grafana. See the how-to article for details:  following blog post for details:
```
Comment (nimbinatus) [inline content/rackspace-metrics/rackspace-metrics-overview.md]: Remove "following blog post for details:"

File: content/rackspace-metrics/rackspace-metrics-overview.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ let us know.
 **Notes:**
 -   The roles needed for accessing the Rackspace Metrics API are
     described at:
-    <https://developer.rackspace.com/docs/metrics/v2/developer-guide/#roles-available-for-rackspace-metrics>
+    <https://developer.rackspace.com/docs/metrics/v2/general-api-info/role-based-access-control/>
```
Comment (nimbinatus) [inline content/rackspace-metrics/rackspace-metrics-overview.md]: remove the hyphens before "The roles..." and add a period after the hyperlink.

File: content/rackspace-metrics/rackspace-metrics-overview.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ let us know.
...
 -   For more information about role-based access control, see [Overview: Role Based Access Control (RBAC)](/how-to/overview-role-based-access-control-rbac).
```
Comment (nimbinatus) [inline content/rackspace-metrics/rackspace-metrics-overview.md]: Remove the hyphen before "For more information..."

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cf688f2b9f/how-to/)

File: content/rackspace-metrics/rackspace-metrics-overview.md (hunk 1 above)
Comment (wolfdancer) [inline content/rackspace-metrics/rackspace-metrics-overview.md]: Fixed.  Thanks.

File: content/rackspace-metrics/rackspace-metrics-overview.md (hunk 2 above)
Comment (wolfdancer) [inline content/rackspace-metrics/rackspace-metrics-overview.md]: Is this what you meant?

**Comment by nimbinatus** [comment]
LGTM, +1

**Pull request merged**